pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotBrokerLogger.java

```diff
@@ -80,4 +88,32 @@ public Map<String, String> setLoggerLevel(@ApiParam(value = "Logger name") @Path
       @ApiParam(value = "Logger level") @QueryParam("level") String level) {
     return LoggerUtils.setLoggerLevel(loggerName, level);
   }
+
+  @GET
+  @Path("/loggers/files")
+  @Produces(MediaType.APPLICATION_JSON)
+  @ApiOperation(value = "Get all local log files")
+  public Set<String> getLocalLogFiles() {
```

Comment:
I guess we should put `@Authenticate(AccessType.READ)` here as well so that not all users have access to the log files.